substrate/frame/broker/src/lib.rs

```diff
@@ -410,6 +420,24 @@ pub mod pallet {
 			/// The core whose workload is no longer available to be renewed for `when`.
 			core: CoreIndex,
 		},
+		/// A new listing has been created.
+		ListingCreated {
+			region_id: RegionId,
+			owner: T::AccountId,
+			price: BalanceOf<T>,
+		},
+		/// A region has been successfully sold.
+		RegionSold {
```

Comment:
A secondary market can already be built by trading CoreTime NFTs, or? Could be traded on KodaDot for example?  
Why do we need additional functionality in here?

Reply:
For the sake of simplicity.